flow-typed/npm/prop-types_v15.x.x.js

```diff
@@ -1,15 +1,15 @@
 /**
- * Copyright (c) 2015-present, Facebook, Inc.
+ * Copyright (c) 2018-present, Facebook, Inc.
```

Comment:
Why change this line ?

Reply:
This line will be updated to have no reference to the year in an upcoming commit. I'd leave it untouched, to avoid merge conflicts...